styles/neovim.io/catppuccin.user.less

```
@@ -0,0 +1,74 @@
+/* ==UserStyle==
+@name neovim.io Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/neovim.io
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/neovim.io
+@version 2000.01.01
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/neovim.io/catppuccin.user.less
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Aneovim.io
+@description Soothing pastel theme for neovim.io
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@import "https://userstyles.catppuccin.com/lib/lib.less";
+
+@-moz-document domain("neovim.io") {
+  @import url("https://unpkg.com/@catppuccin/highlightjs@1.0.0/css/catppuccin-variables.important.css");
+
+  :root {
+    @media (prefers-color-scheme: light) {
+      #catppuccin(@lightFlavor);
+    }
+
+    @media (prefers-color-scheme: dark) {
+      #catppuccin(@darkFlavor);
+    }
+  }
+
+  #catppuccin(@flavor) {
+    #lib.palette();
+    #lib.defaults();
+    #lib.css-variables();
+
+    --fg-color: @text;
+    --inverse-fg-color: @surface0;
+    --bg-color: @base;
+    --accent-bg-color: @mantle;
+    --link-color: @accent;
```

I think since there is already so much accent colored stuff and the docs are very link heavy, it migth make sense to use blue links here. The only tricky part is the text decoration, if you try it out you'll see what I mean.
```suggestion
    --link-color: @blue;
```

---

oh i already know exactly what you mean - ill take a look at what i can do